_dev/apps/ui/src/components/commons/user-not-connected-to-bo-popin.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <ps-modal
+    id="PopinUserNotConnectedToBo"
+    v-bind="$attrs"
+    hide-footer
+    hide-header
+    ref="modal"
+    @shown="startCountdown"
+    @hidden="reload()"
+  >
+    <b-card
+      style=""
+      class="card border-0"
+    >
+      <b-card-body align="left">
```

Comment:
I've been wondering why we had several `align` attributes in your code, and discovered the implementation of this modal is different from the others.

I recommend you reuse the other modals as example (title attribute, slots....) such as _dev/apps/ui/src/components/google-account/google-account-popin-disconnect.vue